.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
   # Run the C benchmark tests. 
   c-benchmark-tests:
-    uses: lf-lang/lingua-franca/.github/workflows/benchmark-tests.yml@master
+    uses: lf-lang/lingua-franca/.github/workflows/benchmark-tests.yml@c-benchmark-runner
```

Comment:
This should be reverted to master. There should be no need to change this unless the `benchmark-tests.yaml` workflow itself is updated in this branch.